ferrocene/ci/configure.sh

```diff
@@ -138,6 +143,36 @@ add --release-description="Ferrocene by Ferrous Systems"
 #                                                                            #
 ##############################################################################
 
+# Set the target used for the build itself (build system, initial compiler
+# stages, etc). This depends on the OS used in CI.
+#
+# If this configuration is missing or changed, the wrong build platform will be
+# used by the build system.
+if [[ "${OSTYPE}" = "msys" ]]; then
+    # In theory, when run under MSYS the default ABI would be GNU, not MSVC.
+    # We're running under MSYS just for compatibility with our CI scripts, not
+    # because we target the GNU ABI, so switching the build platform to the
+    # MSVC one is correct.
+    add --build="x86_64-pc-windows-msvc"
+    add --set target.aarch64-unknown-none.cc=clang
```

Comment:
On Windows, if these are unset it'll try to use `cc` which... isn't a thing Windows usually has.